core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2398,4 +2398,20 @@ package object config {
       .version("3.3.0")
       .intConf
       .createWithDefault(5)
+
+  private[spark] val HEARTBEAT_RECEIVER_CHECK_WORKER_LAST_HEARTBEAT =
+    ConfigBuilder("spark.driver.heartbeat.checkWorkerLastHeartbeat")
+      .internal()
```

Comment:
Add doc for it and mention that this only works for Standalone.

Reply:
Updated https://github.com/apache/spark/pull/37411/commits/02ef67917ac65a088acfaf20e42fe9ef5b823c04